src/Components/Navbar.jsx

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import { Link, useNavigate } from "react-router-dom";
+import SecondaryButton from "./SecondaryButton";
+
+const Navbar = ({ setIsLoggedIn }) => {
+  const navigate = useNavigate(); // Use navigate to change routes
+
+  const handleLogout = () => {
+    setIsLoggedIn(false);
+    navigate("/");
+  };
+
+  return (
+    <nav style={styles.navbar}>
+      
+    <div style={styles.navbarLeft}>
+        <Link to="/home">
+          <img 
+            src="src/MediaFiles/Logo.png" 
+            alt="Logo" 
+            style={styles.logo} />
+        </Link>
+    </div>
+
+    <div style={styles.navbarRight}>
+       <div style={styles.iconContainer}>
+          <Link to="/education">
+            <img 
+              src="src/MediaFiles/ConversionCourses.png" 
+              alt="ConversionCourses" 
+              style={styles.icon} />
+          </Link>
+          <Link to="/people">
+            <img 
+              src="src/MediaFiles/Network.png" 
+              alt="Network" 
+              style={styles.icon} />
+          </Link>
+          <Link to="/jobs">
+            <img 
+              src="src/MediaFiles/Jobs.png" 
+              alt="Jobs" 
+              style={styles.icon} />
+          </Link>
+          <Link to="/messages">
+            <img
+              src="src/MediaFiles/Messages.png"
+              alt="Messages"
+              style={styles.icon}
+            />
+          </Link>
+            <img
+              src="src/MediaFiles/Notifications.png"
+              alt="Notifications"
+              style={styles.icon}
+            />
+          <Link to="/profile">
+            <img 
+              src="src/MediaFiles/Profile.png" 
+              alt="Profile" 
+              style={styles.profileImage} />
+          </Link>
+
+        <input 
+          type="text" 
+          placeholder="Start typing..." 
+          style={styles.searchBar}
+        />
+
+        <SecondaryButton 
+          onClick={handleLogout}>
+            Log Out
+        </SecondaryButton>
+
+      </div>
+    </div>
+  </nav>
+  );
+};
+
+
+// Styling
+const styles = {
+  navbar: {
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+    padding: "10px 20px",
+    width: "100vw",
+    boxSizing: "border-box",
+    backgroundColor: "#ffffff",
+    boxShadow: "0 2px 5px rgba(0, 0, 0, 0.1)",
+    position: "fixed",
+    top: 0,
+    left: 0,
+    zIndex: 1000,
```

Comment:
do you really need this? Is it because of other high zIndex elements? You might want to read up on this